packages/http/libraries/core/src/models/RouterParams.ts

```diff
@@ -0,0 +1,9 @@
+import { ControllerMethodMetadata } from './ControllerMethodMetadata';
+import { RequestHandler } from './RequestHandler';
+
+export type RouterParams<TRequest, TResponse, TNextFunction> = Omit<
+  ControllerMethodMetadata,
+  'methodKey'
+> & {
+  handler: RequestHandler<TRequest, TResponse, TNextFunction>;
+};
```

Comment:
I would prefer to avoid `Omit` as much as possible.

Reply:
Fixed